easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.8.2-2023a.yml

```diff
@@ -12,3 +12,4 @@ easyconfigs:
       options:
         from-pr: 19270
   - SciPy-bundle-2023.07-gfbf-2023a.eb
+  - OpenFOAM-10-foss-2023a.eb
```

Comment:
There will be a merge conflict here once https://github.com/EESSI/software-layer/pull/386 gets merged (but in terms of stuff to install, it shouldn't matter), should be trivial to resolve though.

Reply:
Indeed it happened, just fixed the merge conflict :)